test/unit/concerns/background_deletion_test.rb

```diff
@@ -36,5 +36,20 @@ def test_delete_integration
       DeleteObjectHierarchyWorker.expects(:perform_later).with(user, anything, 'destroy').never
       DeleteObjectHierarchyWorker.perform_now(double_object)
     end
+
+    test 'all associations of the config can be constantized' do
+      Rails.application.eager_load!
+
+      models = ActiveRecord::Base.descendants - [DoubleObject, DoubleActiveRecordObject, SecondDoubleActiveRecordObject]
+
+      models.each do |klass|
+        next unless klass.included_modules.include? BackgroundDeletion
```

Comment:
They will always include it no? Because it is included in ApplicationRecord

Reply:
Ouch, right!
![image](https://user-images.githubusercontent.com/11318903/74656389-5155ba00-518e-11ea-9592-22b38bbfbb6b.png)